### ISSUE TITLE
Chrome showing weird outputs

### ISSUE THREAD

**Issue by Athelena** [cityjson/ninja#79]
There seems to be an issue in Chrome where the output has a weird triangulation and spikes. This doesn't seem to be an issue in Safari or Firefox.

![image](https://user-images.githubusercontent.com/19203377/195840620-8a859b20-d5d9-4dc0-ae13-66b760db01ee.png)

![image](https://user-images.githubusercontent.com/19203377/195840677-6ab1e5c9-65b8-423a-bba0-840a20b86a77.png)

![image](https://user-images.githubusercontent.com/19203377/195840846-e2cc1884-cf97-4038-ac38-e3ce4b3a3414.png)

**Comment by Athelena** [comment]
Turns out the issue is old hardware.

**Comment by liberostelios** [comment]
Indeed, seems like we won't be able to solve this and it won't worth the investment.

Thank you so much for flagging this. 🙏
